url6da19b5a7a_krakrak_Adventure_Game_tgz-pJ8-adventure_MainJ8/src/commands/Use.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package commands;
+
+import items.Item;
+import items.Useable;
+import java.util.List;
+
+/**
+ * Command that allows you to use the specified Item
+ * @author Irakiza Krak
+ */
+public class Use extends AbstractCommand{
+    private boolean contains;
+
+    public Use(){
+        super("USE", "Allows you to use an useable object", "use");
+    }
+    @Override
+    public void execute(Game game, String params) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 15. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.